noxfile.py

```diff
@@ -755,23 +763,56 @@ def notebook(session: nox.Session):
             CURRENT_DIRECTORY / "scripts" / "notebooks_fill_params.py",
             *notebooks,
         )
+        processes = []
+        manager = multiprocessing.Manager()
```

Comment:
The `manager` is only used to create the `error_flag`, not in running the processes?